x/rollapp/client/cli/tx_create_rollapp.go

```diff
@@ -12,25 +14,30 @@ import (
 
 func CmdCreateRollapp() *cobra.Command {
```

Comment:
@zale144 I guess it's better to add `long` description with possible fields and expected values